src/z3c/rml/attr.py

```diff
@@ -553,10 +562,13 @@ def fromUnicode(self, value):
         try:
             return reportlab.lib.colors.toColor(value)
         # Bare except, since code raises string exception: Invalid color value
-        except:
+        except:  # noqa
```

Comment:
Oh, I just checked, it now raises a proper `ValueError`. We should fix it in this case.